pkgs/development/libraries/srt/default.nix

```
@@ -14,7 +19,15 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  buildInputs = [ openssl ];
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.hostPlatform.isMinGW [
+    windows.mingw_w64_pthreads
+  ];
+
+  patches = lib.optionals stdenv.hostPlatform.isMinGW [
```

Can this patch be included unconditionally?

```suggestion
  patches = [
```

---

If anything, it also should be upstreamed.